_posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md

```diff
@@ -0,0 +1,643 @@
+---
+layout: post
+authors: [yannick_de_turck]
+title: "Lagom 1.4 and Kubernetes orchestration"
+image: /img/lagom-kubernetes.png
+tags: [Lagom, Java, Scala, Reactive, Kubernetes, Minikube]
+category: Orchestration
+comments: true
+---
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Upgrading to Lagom 1.4](#upgrading-to-lagom-14)
+3. [Lightbend's orchestration tools](#lightbends-orchestration-tools)
+4. [Adding Kubernetes support to our Lagom Shop Scala application](#adding-kubernetes-support-to-our-lagom-shop-scala-application)
+5. [Building and publishing Docker images](#building-and-publishing-docker-images)
+6. [Locally deploying to Kubernetes using Minikube](#locally-deploying-to-kubernetes-using-minikube)
+7. [Generating Kubernetes resources for production](#generating-kubernetes-resources-for-production)
+8. [Conclusion](#conclusion)
+9. [Extra resources](#extra-resources)
+
+## Introduction
+In this blog post we will take a closer look at the [Lightbend Orchestration tools](https://developer.lightbend.com/docs/lightbend-orchestration-kubernetes/latest/index.html){:target="_blank" rel="noopener noreferrer"}.
+Tools helping you deploy your Lagom application to Kubernetes and DC/OS.
+It was already possible to deploy Lagom application to Kubernetes as [this guide](https://developer.lightbend.com/guides/lagom-kubernetes-k8s-deploy-microservices/){:target="_blank" rel="noopener noreferrer"} demonstrates but it involved more manual tasks and having written the Kubernetes resource and configuration files yourself, as it usually goes.
```

Comment:
as ... demonstrates, but it involved more manual tasks and having to write the Kubernetes ...